### PR TITLE
Implementation of write-only Run.track()

### PR DESCRIPTION
**Pull request by aramaim**

### CODE REVIEW

**Comment by aramaim** [comment]
resolves #1227

File: aim/sdk/run.py [hunk 1]
```diff
@@ -636,6 +608,77 @@ def _get_sequence(
         sequence = seq_cls(sequence_name, context, self)
         return sequence if bool(sequence) else None
 
+    def _prepare_sequence_info(self, read_only):
+        if read_only:
+            return
+
+        try:
+            for ctx_id in self.meta_tree['contexts'].keys():
```
Comment (alberttorosyan) [inline aim/sdk/run.py:616]: please check if its possible to use `self.meta_run_tree.subtree('traces').items() to iterate over contexts/traces.

File: aim/sdk/run.py [hunk 2]
```diff
@@ -407,50 +410,19 @@ def _track_impl(
...
-        if seq_info.sequence_dtype is not None:
-            def update_trace_dtype(new_dtype):
-                self.meta_tree['traces_types', new_dtype, ctx.idx, name] = 1
-                seq_info.sequence_dtype = self.meta_run_tree['traces', ctx.idx, name, 'dtype'] = new_dtype
-
-            compatible = check_types_compatibility(dtype, seq_info.sequence_dtype, update_trace_dtype)
-            if not compatible:
-                raise ValueError(f'Cannot log value \'{value}\' on sequence \'{name}\'. Incompatible data types.')
-
+        seq_info = self._get_sequence_info(ctx, name)
```
Comment (alberttorosyan) [inline aim/sdk/run.py:420]: `_get_or_create_sequence_info`?

Comment (aramaim) [reply aim/sdk/run.py:420]: fixed

File: aim/sdk/run.py (hunk 1 above)
Comment (aramaim) [reply aim/sdk/run.py:616]: good point. yes, we can use `self.meta_run_tree.get('traces', {}).items()`

**Review by alberttorosyan — APPROVED** [review]
looks great!

**Pull request merged**